lib/heroku/command/pg.rb

```diff
@@ -726,19 +736,6 @@ def wait_for(attach, interval)
     end
   end
 
-  def find_uri
-    return @uri if defined? @uri
-
-    attachment =  generate_resolver.resolve(shift_argument, "DATABASE_URL")
-    if attachment.kind_of? Array
```

Comment:
I don't really understand this type switch very well. It seems incongruous with other cases with how generate_resolver.resolve is used, so is it dead code?